astropy/stats/info_theory.py

```diff
@@ -176,7 +176,7 @@ def bayesian_info_criterion_lsq(ssr, n_params, n_samples):
     >>> bic_t = bayesian_info_criterion_lsq(ssr_t, 4, x.shape[0])
     >>> bic_g = bayesian_info_criterion_lsq(ssr_g, 3, x.shape[0])
     >>> bic_t - bic_g # doctest: +FLOAT_CMP
-    30.644474706065466
+    212.83039990507456
```

Comment:
Is it okay to just change the value like that? Maybe someone wants to check if the answer is indeed correct? @crawfordsm ?